### PR TITLE
Add missing space to warning message

### PR DESCRIPTION
**Pull request by The-Compiler**
This was introduced in GH-13409 when changing the message text.

cc @Carreau 

This should probably go to the 3.8 branch as well?

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

**Review by Carreau — APPROVED** [review]

**Comment by Carreau** [comment]
Oops, thanks !

File: Lib/collections/__init__.py
Comment (aeros) [inline Lib/collections/__init__.py:51]: ~~This might not be necessary, but it looks like it's also missing a period at the end.~~

**Review by aeros — APPROVED** [review]

**Comment by aeros167** [comment]
@The-Compiler Thanks for the contribution! Yes, this should be backported to 3.8 as well since the space is also missing on that branch.

**Comment by The-Compiler** [comment]
FWIW I signed the CLA right after opening this PR, but looks like it hasn't been received yet?

File: Lib/collections/__init__.py
Comment (The-Compiler) [reply Lib/collections/__init__.py:51]: FWIW looks like most warning messages (`grep -R -F -A2 'warnings.warn(' Lib`) don't have a trailing dot.

**Comment by aeros167** [comment]
@The-Compiler CLA approval usually takes around 24 hours. Sometimes though you have to manually refresh the status on GitHub, use the link from @the-knights-who-say-ni: [check yourself](https://check-python-cla.herokuapp.com/).

File: Lib/collections/__init__.py
Comment (aeros) [reply Lib/collections/__init__.py:51]: Oh my mistake then, feel free to disregard that suggestion.

**Comment by The-Compiler** [comment]
I'm aware - it's been three days and the checker still says "The-Compiler has not signed CLA".

**Comment by aeros167** [comment]
@The-Compiler The only other option that I'm aware of is attempting to send email to contributors@python.org

**Comment by The-Compiler** [comment]
@aeros167 Thanks, that helped - turns out I somehow have two bugs.python.org accounts ("The-Compiler" from 2012, "The Compiler" from 2014) but never noticed, and they accidentally marked the wrong one.

Anything else I need to do at this point? I don't think this needs a news item or issue number FWIW.

**Comment by aeros167** [comment]
@The-Compiler At this point, it's just waiting for a core dev review, they can add a skip news and skip issue label for you

**Review by rhettinger — APPROVED** [review]

**Pull request merged**
